domains/jenin.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "jeninh",
+    "email": "cubingjulian@gmail.com"
+  },
+  "record": {
+    "CNAME": "jeninh.github.io"
```

Comment:
Site needs to be completed per our requirements:

![image](https://github.com/user-attachments/assets/707ba9e1-3b60-4cff-9ba9-54fc0b771252)